Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -2696,6 +2721,6 @@ extension SignalProducer {
 			}
 		}
 
-		return SignalProducer.attempt(operation)
+		return SignalProducer(operation)
```

Comment:
Maybe add `attempt` as a label? Having so many unnamed initializers leads to ambiguity.